otelcli/root.go

```diff
@@ -67,6 +67,10 @@ func addClientParams(cmd *cobra.Command) {
 	cmd.Flags().StringToStringVar(&config.Headers, "otlp-headers", defaults.Headers, "a comma-sparated list of key=value headers to send on OTLP connection")
 	cmd.Flags().BoolVar(&config.Blocking, "otlp-blocking", defaults.Blocking, "block on connecting to the OTLP server before proceeding")
 
+	cmd.Flags().StringVar(&config.Certificate, "cacert", defaults.Certificate, "a file containing the server's CA certificate")
+	cmd.Flags().StringVar(&config.ClientCert, "client-cert", defaults.ClientCert, "a file containing the client certificate")
+	cmd.Flags().StringVar(&config.ClientKey, "client-key", defaults.ClientKey, "a file containing the client certificate key")
```

Comment:
^^ need feedback on these. Do they look ok? Should there be a set of --tls-* options?

Reply:
I made the change, it looks better this way:

```
      --tls-ca-cert string            a file containing the certificate authority bundle
      --tls-client-cert string        a file containing the client certificate
      --tls-client-key string         a file containing the client certificate key
      --tls-no-verify                 insecure! disables verification of the server certificate and name, mostly for self-signed CAs
      --no-tls-verify                 (deprecated) same as --tls-no-verify
```